plugins/destiny.py

```diff
@@ -218,6 +243,21 @@ def xur(text, bot):
         return output
 
 
+@hook.command('lore')
+def lore(bot):
+    if not LORE_CACHE:
+        global LORE_CACHE
+        LORE_CACHE = get("{}/Vanguard/Grimoire/Definition/".format(BASE_URL),
+            headers=HEADERS).json()['Response']['themeCollection']
+    level1 = randint(1, len(LORE_CACHE)) - 1
+    level2 = randint(1, len(LORE_CACHE[level1]['pageCollection'])) - 1
+    level3 = randint(1, len(LORE_CACHE[level1]['pageCollection'][level2]['cardCollection'])) - 1
+    card = LORE_CACHE[level1]['pageCollection'][level2]['cardCollection'][level3]
```

Comment:
each level has a variable number of items. i suspect some may be blank or missing, resulting in blank or errors being returned occasionally.